src/proof/clause_id.h

```diff
@@ -28,6 +28,12 @@ namespace CVC4 {
  */
 typedef unsigned ClauseId;
 
+const ClauseId ClauseIdEmpty(-1);
```

Comment:
What are these? Can you add a some quick (basic) documentation here? E.g. "Temporary, to do delete when X" or whatever.

Reply:
These are used to control registration of clauses in the old sat proof (which is used for unsat cores, so we have to keep it for now). I'll add a comment. As a rule of thumb, everything that is in `/proof` currently will be deleted in its current form.